examples/imdb_bidirectional_lstm.py

```diff
@@ -39,24 +38,20 @@
 y_test = np.array(y_test)
 
 print('Build model...')
-model = Graph()
-model.add_input(name='input', input_shape=(maxlen,), dtype=int)
-model.add_node(Embedding(max_features, 128, input_length=maxlen),
-               name='embedding', input='input')
-model.add_node(LSTM(64), name='forward', input='embedding')
-model.add_node(LSTM(64, go_backwards=True), name='backward', input='embedding')
-model.add_node(Dropout(0.5), name='dropout', inputs=['forward', 'backward'])
-model.add_node(Dense(1, activation='sigmoid'), name='sigmoid', input='dropout')
-model.add_output(name='output', input='sigmoid')
+model = Sequential()
+model.add(Embedding(max_features, 128, input_length=maxlen))
+model.add(Bidirectional(LSTM(64)))
+model.add(Dropout(0.5))
+model.add(Dense(1, activation='sigmoid'))
```

Comment:
I wrote
<code>
model.add(Dense(output_dim=entity_size, init="normal", activation='sigmoid')))
</code>
And an error occurs:
<code>
ValueError: dimension mismatch in args to gemm (128,256)x(512,484)->(128,484)
</code>
Note:entity_size=484,LSTM hidden_size=256